From 6b243e02a266df3be80f899d182f10d83a4ea577 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Marco=20Trevisan=20=28Trevi=C3=B1o=29?= <mail@3v1n0.net>
Date: Fri, 13 Mar 2020 19:13:29 +0100
Subject: [PATCH 17/37] docking: Don't sync overview's iconSize anymore

Dash icon size is now a read-only property that uses the dash value, so
there's no need to sync this value, given we would always read the same
value.
---
 docking.js | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/docking.js b/docking.js
index ab4be65..79b1543 100644
--- a/docking.js
+++ b/docking.js
@@ -274,11 +274,6 @@ var DockedDash = GObject.registerClass({
             this._intellihide,
             'status-changed',
             this._updateDashVisibility.bind(this)
-        ], [
-            // Keep dragged icon consistent in size with this dash
-            this.dash,
-            'icon-size-changed',
-            () => { Main.overview.dash.iconSize = this.dash.iconSize; }
         ], [
             // sync hover after a popupmenu is closed
             this.dash,
@@ -321,15 +316,6 @@ var DockedDash = GObject.registerClass({
                 Main.overview.viewSelector._showAppsButton,
                 'notify::checked',
                 this._syncShowAppsButtonToggled.bind(this)
-            ], [
-                // This duplicate the similar signal which is in owerview.js.
-                // Being connected and thus executed later this effectively
-                // overwrite any attempt to use the size of the default dash
-                //which given the customization is usually much smaller.
-                // I can't easily disconnect the original signal
-                Main.overview.dash,
-                'icon-size-changed',
-                () => { Main.overview.dash.iconSize = this.dash.iconSize; }
             ]);
         }
 
@@ -1795,9 +1781,6 @@ var DockManager = class DashToDock_DockManager {
         Main.overview.dash.set_width(-1); //reset default dash size
         // This force the recalculation of the icon size
         Main.overview.dash._maxHeight = -1;
-
-        // reset stored icon size  to the default dash
-        Main.overview.dash.iconSize = Main.overview.dash.iconSize;
     }
 
     _onShowAppsButtonToggled(button) {
-- 
2.26.0

